tests/test_reprojection_utils.py

```diff
@@ -53,16 +53,58 @@ def test_parallax_equinox(self):
         npt.assert_almost_equal(corrected_coord1.ra.value, expected_ra)
         npt.assert_almost_equal(corrected_coord1.dec.value, expected_dec)
 
-        corrected_coord2 = correct_parallax(
-            coord=sc2,
-            obstime=time2,
-            point_on_earth=loc,
-            guess_distance=50.0,
+        corrected_coord2, _ = correct_parallax(
+            coord=self.sc2,
+            obstime=self.icrs_time2,
+            point_on_earth=self.eq_loc,
+            heliocentric_distance=50.0,
         )
 
         npt.assert_almost_equal(corrected_coord2.ra.value, expected_ra)
         npt.assert_almost_equal(corrected_coord2.dec.value, expected_dec)
 
+        assert type(corrected_coord1) == SkyCoord
```

Comment:
I think the Python style is generally `type(variable) is something` instead of `==`